examples/networking/sttp/src/Main.scala

```diff
@@ -0,0 +1,94 @@
+package example
+
+import cats.effect.IO
+import tyrian.Html.*
+import tyrian.*
+import sttp.client4.impl.cats.FetchCatsBackend
+import sttp.client4.*
+import sttp.client4.circe.*
+import io.circe.generic.auto.*
+import cats.effect.kernel.Async
+import cats.implicits.*
+
+import scala.scalajs.js.annotation.*
+
```

Comment:
If you add this here, it will appear in the docs (doesn't need to be in the integration example version, but does no harm either):

```scala
/** ## How to call timeapi.io with the sttp HTTP client
  *
  * In this example, we'll see how to wrap the sttp HTTP client in a Cmd, so that we can call timeapi.io to obtain time information about our IP address.
  */
```